ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -67,6 +67,10 @@ public void Perform()
                 AssemblyDefinition attributeAsm = AssemblyDefinition.ReadAssembly(_options.AttributeFile, new ReaderParameters(ReadingMode.Immediate) { AssemblyResolver = _repackContext.GlobalAssemblyResolver });
                 _repackCopier.CopyCustomAttributes(attributeAsm.CustomAttributes, targetAssemblyDefinition.CustomAttributes, null);
                 _repackCopier.CopyCustomAttributes(attributeAsm.CustomAttributes, targetAssemblyMainModule.CustomAttributes, null);
+                if (_options.Version == null)
```

Comment:
Idea sounds good (so does compat with ILMerge), but couple of (late) remarks here:
- this step is performed pretty late in the process, and the asembly fully qualified name may have already been used prior to its execution. It should probably be moved earlier (e.g. the attr assembly could be loaded in the ctor, and the version applied in ILRepack.cs instead before any repack step happens)
- if no version is provided in the attr file, it'll override the original assembly version (if it had one), is that intended (/compatible with ILMerge) - doesn't seem ok to me